Makefile

```diff
@@ -1,4 +1,4 @@
 install: ${GOBIN}/makex
 
-${GOBIN}/makex: $(shell find -type f -and -name '*.go')
+${GOBIN}/makex: $(shell /usr/bin/find -type f -and -name '*.go')
```

Comment:
Is this necessary for Windows? On some *nix systems, this might be located at `/usr/local/bin/find` or elsewhere.